server/go.mod

```diff
@@ -31,6 +31,7 @@ require (
 	go.etcd.io/etcd/client/v2 v2.306.0-alpha.0
 	go.etcd.io/etcd/client/v3 v3.6.0-alpha.0
 	go.etcd.io/etcd/pkg/v3 v3.6.0-alpha.0
+	go.etcd.io/gofail v0.2.0
```

Comment:
I don't have an intuition for how undesirable it might be to add this dependency to a core package like `server`